src/app/pages/apptrust/apptrust.page.html

```diff
@@ -9,6 +9,48 @@
   </ion-toolbar>
 </ion-header>
 
-<ion-content >
-  
-</ion-content>
+<ion-content>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>Access the Mobile Security Service</ion-card-title>
+    </ion-card-header>
+    <ion-card>
+      <ion-item>
+        <a href="{{serverUrl.url}}">Mobile Security Service</a>
+      </ion-item>
+    </ion-card>
+  </ion-card>
+  <ion-card>
+    <ion-card-header>
+      <ion-card-title>Disable Application</ion-card-title>
+      <ion-card-subtitle>In the Mobile Security Service disable the following Application </ion-card-subtitle>
+    </ion-card-header>
+
+    <ion-item>
+      <a>org.aerogear.ionic.showcase</a>
```

Comment:
Should this be a link?

Reply:
no , but was an easy way to get it to render in a different colour